sources/lib/util/options.ml

```diff
@@ -148,6 +164,29 @@ let vsave_used_context = ref false
 let vprofiling_period = ref 0.
 let vprofiling = ref false
 
+let match_input_format f : input_format =
+  match f with
+  | "native" | "altergo" | "alt-ergo" -> Native
+  | "smtlib2" | "smt-lib2" -> Smtlib2
+  | "why3" -> Why3
+  (* | "szs" | "SZS" -> SZS *)
+  | s -> Unknown(s)
+
+let match_output_format f : output_format =
+  match f with
+  | "native" | "altergo" | "alt-ergo" -> Native
+  | "smtlib2" | "smt-lib2" -> Smtlib2
+  | "why3" -> Why3
+  (* | "szs" | "SZS" -> "szs" *)
+  | s -> Unknown(s)
+
+let match_extension e =
+  match e with
+  | "ae" | "why" | "mlw" -> "native"
+  | "smt2" | "psmt2" -> "smtlib"
+  | "why3" -> "why3"
+  (* | "szs" | "SZS" -> "szs" *)
+  | s -> s
```

Comment:
This should return an `input_format` and not a string. Basically, if you later want to match on a value returned by a function (which is the case with this function), strings are a bad idea...